docs/content-editing/add-a-search-block.md

```diff
@@ -0,0 +1,431 @@
+
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-28.jpeg
+    :alt: Select `Date Range` to display date picker widgets for a modification date range.
+    :target: /_images/add-a-search-block-step-28.jpeg
+    ```
+    +++
+    _Select {guilabel}`Date Range` to display date picker widgets for a modification date range._
+    ````
+
+1.  Click on "Sort on options".
+
+    This adds a widget with options for the user to sort their search.
+    
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-29.jpeg
+    :alt: Click on "Sort on options".
+    :target: /_images/add-a-search-block-step-29.jpeg
+    ```
+    +++
+    _Click on "Sort on options"._
+    ````
+
+1.  Click "Creation date" and "Effective date" to add them to the list of options to sort by.
+
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-30.jpeg
+    :alt: Click "Creation date" and "Effective date" to add them to the list of options to sort by.
+    :target: /_images/add-a-search-block-step-30.jpeg
+    ```
+    +++
+    _Click "Creation date" and "Effective date" to add them to the list of options to sort by._
+    ````
+
+1.  {guilabel}`Save` your page.
+
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-31.jpeg
+    :alt: **[[Save]]** your page.
+    :target: /_images/add-a-search-block-step-31.jpeg
+    ```
+    +++
+    _{guilabel}`Save` your page._
+    ````
+
+1.  The search field will allow the user to search with a text search.
+
+    The facets will narrow that search down.
+
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-32.jpeg
+    :alt: The search field will allow the user to search with a text search.
+    :target: /_images/add-a-search-block-step-32.jpeg
+    ```
+    +++
+    _The search field will allow the user to search with a text search._
+    ````
+
+1.  The Tags facet will allow the user to select items that are tagged and filter according to that tag.
+
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-33.jpeg
+    :alt: The Tags facet will allow the user to select items that are tagged and filter according to that tag.
+    :target: /_images/add-a-search-block-step-33.jpeg
+    ```
+    +++
+    _The Tags facet will allow the user to select items that are tagged and filter according to that tag._
+    ````
+
+1.  The Date range facet will allow the user to narrow down their search inside of a specific date range.
+
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-34.jpeg
+    :alt: The Date range facet will allow the user to narrow down their search inside of a specific date range.
+    :target: /_images/add-a-search-block-step-34.jpeg
+    ```
+    +++
+    _The Date range facet will allow the user to narrow down their search inside of a specific date range._
```

Comment:
```suggestion
1.  The {guilabel}`Date range` facet will allow the user to narrow down their search inside of a specific date range.

    ````{card}
    ```{image} /content-editing/_static/add-a-search-block-step-34.jpeg
    :alt: The `Date range` facet will allow the user to narrow down their search inside of a specific date range.
    :target: /_images/add-a-search-block-step-34.jpeg
    ```
    +++
    _The {guilabel}`Date range` facet will allow the user to narrow down their search inside of a specific date range._
```